lib/signalfx/tracing/version.rb

```diff
@@ -1,5 +1,5 @@
 module Signalfx
   module Tracing
-    VERSION = "1.2.0"
+    VERSION = "1.3.0"
```

Comment:
nit: wouldn't we want to change the version upon release?  We don't use `x.y.z-dev` to indicate transitional work though, so I see why you did this.